pkg/kube/secretcontroller/secretcontroller.go

```diff
@@ -296,6 +318,13 @@ func (c *Controller) HasSynced() bool {
 	return synced
 }
 
+func (c *Controller) Clusters() map[string]map[cluster.ID]*Cluster {
+	if c == nil {
```

Comment:
why would `c` be `nil`?

Reply:
It should be impossible, but I think you can hit debug endpoints before this stuff gets fully initialized.